_episodes_rmd/04-data-structures-part1.Rmd

```diff
@@ -376,6 +376,26 @@ These make sense. But what about
 str(cats$coat)
 ```
 
+> ## Tip: Renaming dataframe columns
+>
+> Dataframes have column names we can access them like we did with vectors
+>
+> ```{r}
+> names(cats)
+> ```
+>
+> If you want to rename the second column you can assign the new name to the second element of names
```

Comment:
Grammar: "If you want to rename the second column you can assign the new name to the second element of names" -> "If you want to rename the second column, you can assign a new name to the second element of `names(cats)`."